plugins/BEdita/API/tests/TestCase/Controller/ObjectsControllerTest.php

```diff
@@ -128,12 +129,14 @@ public function testIndex()
                             'links' => [
                                 'related' => 'http://api.example.com/documents/2/test',
                                 'self' => 'http://api.example.com/documents/2/relationships/test',
+                                'available' => 'http://api.example.com/objects?filter%5Btype%5D%5B0%5D=documents&filter%5Btype%5D%5B1%5D=profiles',
```

Comment:
It's that version, only URL-encoded. Also, due to some reason, PHP likes to encode array keys as well when building query strings, thus it becomes `filter[type][0]=documents&filter[type][1]=profiles`.